packages/rancher-gatekeeper/rancher-gatekeeper.patch

```diff
@@ -1,26 +1,28 @@
 diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-gatekeeper/charts-original/Chart.yaml packages/rancher-gatekeeper/charts/Chart.yaml
 --- packages/rancher-gatekeeper/charts-original/Chart.yaml
 +++ packages/rancher-gatekeeper/charts/Chart.yaml
-@@ -1,6 +1,6 @@
+@@ -1,10 +1,17 @@
  apiVersion: v1
- description: A Helm chart for Gatekeeper
+ appVersion: v3.1.0
+-description: A Helm chart for Gatekeeper
++description: Policy-based control for cloud native environments
+ home: https://github.com/open-policy-agent/gatekeeper
+ keywords:
+-- open policy agent
 -name: gatekeeper
++  - open policy agent
++  - security
 +name: rancher-gatekeeper
- keywords:
-   - open policy agent
- version: v3.1.0-rc.1
-@@ -8,3 +8,11 @@
  sources:
-   - https://github.com/open-policy-agent/gatekeeper.git
- appVersion: v3.1.0-rc.1
+-- https://github.com/open-policy-agent/gatekeeper.git
++  - https://github.com/open-policy-agent/gatekeeper.git
+ version: v3.1.0
 +icon: https://charts.rancher.io/assets/logos/gatekeeper.svg
 +annotations:
 +  catalog.cattle.io/certified: rancher
-+  catalog.cattle.io/experimental: true
```

Comment:
just double checking; did you intend for this annotation to be removed?

Reply:
Yes, this is their first GA release.